<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerReformatModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerReformatModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reformat</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="ctkCheckBox" name="VisibilityCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Slice:</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
       <property name="indicatorIcon">
        <iconset resource="../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
         <normaloff>:/Icons/VisibleOn.png</normaloff>
         <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
       </property>
       <property name="indicatorIconSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="SliceNodeSelector">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLSliceNode</string>
        </stringlist>
       </property>
       <property name="showHidden">
        <bool>true</bool>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ShowReformatWidgetToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show reformat widget in 3D view</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
         <normaloff>:/Icons/SliceWidgetOff.png</normaloff>
         <normalon>:/Icons/SliceWidgetOn.png</normalon>:/Icons/SliceWidgetOff.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="DisplayEditCollapsibleWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display &amp;&amp; Edit</string>
     </property>
     <property name="contentsLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="ctkCollapsibleGroupBox" name="OffsetSlidersGroupBox">
        <property name="title">
         <string>Offset</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="qMRMLLinearTransformSlider" name="OffsetSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <double>-200.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QWidget" name="MinMaxWidget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="OriginCoordinatesGroupBox">
        <property name="title">
         <string>Origin</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QGroupBox" name="CoordinateReferenceGroupBox">
           <property name="title">
            <string>Coordinate Reference</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="OnPlaneRadioButton">
              <property name="text">
               <string>On Plane</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="InVolumeRadioButton">
              <property name="text">
               <string>In Volume</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="OnPlaneGroupBox">
           <layout class="QHBoxLayout" name="OnPlaneLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ctkDoubleSpinBox" name="OnPlaneXdoubleSpinBox"/>
            </item>
            <item>
             <widget class="ctkDoubleSpinBox" name="OnPlaneYdoubleSpinBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="CenterPushButton">
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="ctkCoordinatesWidget" name="InVolumeCoordinatesWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="RotationSlidersGroupBox">
        <property name="title">
         <string>Orientation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="11" column="1">
          <widget class="qMRMLLinearTransformSlider" name="RotateZSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Rotate the slice in the current slice plane, around the slice normal</string>
           </property>
           <property name="minimum">
            <double>-200.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="PALabel">
           <property name="text">
            <string>Vertical:</string>
           </property>
           <property name="indent">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="ISLabel">
           <property name="text">
            <string>In-Plane:</string>
           </property>
           <property name="indent">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="NormalToLRPushButton">
             <property name="text">
              <string>Normal to LR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NormalToPAPushButton">
             <property name="text">
              <string>Normal to PA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NormalToISPushButton">
             <property name="text">
              <string>Normal to IS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="RotateToVolumePlanePushButton">
           <property name="toolTip">
            <string>Rotate the slice view to be aligned with the axes of the displayed volume</string>
           </property>
           <property name="text">
            <string>Rotate to volume plane</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="Normal_2">
           <property name="text">
            <string>Rotation:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="qMRMLLinearTransformSlider" name="RotateYSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Rotate around the slice vertical axis</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-200.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="qMRMLLinearTransformSlider" name="RotateXSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Rotate around the slice horizontal axis</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-200.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Resetlabel">
           <property name="text">
            <string>Reset to:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Normal">
           <property name="text">
            <string>Normal:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="LRLabel">
           <property name="text">
            <string>Horizontal:</string>
           </property>
           <property name="indent">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="12" column="0" colspan="2">
          <widget class="QWidget" name="MinMaxWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkComboBox" name="SliceOrientationSelector">
           <property name="toolTip">
            <string>Slice orientation (Axial, Sagittal, Coronal, Reformat).</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Axial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sagittal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Coronal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reformat</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="13" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1">
          <widget class="ctkCheckablePushButton" name="NormalToCameraCheckablePushButton">
           <property name="text">
            <string>Normal to Camera</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="buttonTextAlignment">
            <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignJustify|Qt::AlignVCenter</set>
           </property>
           <property name="indicatorAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="checkState">
            <enum>Qt::Unchecked</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="ctkCoordinatesWidget" name="NormalCoordinatesWidget">
           <property name="normalized">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="FlipHorizontalPushButton">
             <property name="toolTip">
              <string>Flip slice view horizontally (left-right)</string>
             </property>
             <property name="text">
              <string>Flip H</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="FlipVerticalPushButton">
             <property name="toolTip">
              <string>Flip slice view vertically (upside-down)</string>
             </property>
             <property name="text">
              <string>Flip V</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RotateClockwisePushButton">
             <property name="toolTip">
              <string>Rotate slice view clockwise by 90 degrees</string>
             </property>
             <property name="text">
              <string>Rotate CW</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RotateCounterClockwisePushButton">
             <property name="toolTip">
              <string>Rotate slice view counterclockwise by 90 degrees</string>
             </property>
             <property name="text">
              <string>Rotate CCW</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionLockNormalToCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock Normal To Camera</string>
   </property>
   <property name="toolTip">
    <string>Lock reformat widget's normal to the camera one. </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>ctkPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLLinearTransformSlider</class>
   <extends>qMRMLSliderWidget</extends>
   <header>qMRMLLinearTransformSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerReformatModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SliceNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliceNodeSelector</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>DisplayEditCollapsibleWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
